src/device/device.hpp

```diff
@@ -52,7 +52,7 @@
 #define USE_DEVICE_LEDGER 1
 #endif
 
-#if !defined(HAVE_PCSC) 
+#if !defined(HAVE_HIDAPI) 
 #undef  USE_DEVICE_LEDGER
 #define USE_DEVICE_LEDGER 0
 #endif
```

Comment:
This seems to set USE_DEVICE_LEDGER to 0 or 1, but the code above uses #ifdef, not #if